runtime/string_functions.cpp

```diff
@@ -892,7 +891,7 @@ string f$pack(const string &pattern, const array<mixed> &a) {
             case 'P':
             case 'Q': {
               // stored in the host machine order by the default (Q flag)
-              unsigned long long value_byteordered = ULong(arg.to_string()).l;
+              unsigned long long value_byteordered = static_cast<unsigned long long>(arg.to_string().to_int());
```

Comment:
1) I'm not sure if `string::to_int()` parse string with large unsigned integer correct, please check.
2) Types ULong and Long check overflow and fire warning, but `string::to_int()` doesn't.

Reply:
How would you offer to fix this? Or just to ignore?

Reply:
I have no idea if it matters. Lets ignore.